### ISSUE TITLE
No way to clear search terms

### ISSUE THREAD

**Issue by 0x199x** [ezra-bible-app/ezra-bible-app#865]
Hi. I think this is a bug unless I'm missing something obvious

I don't see any way to clear search terms.  

For example, search for "Israel", all instances are highlighted.  Erase the search text and Israel is still highlighted.  Click the "Search" button with empty search text and nothing happens, Israel is still highlighted.  Erase the search box text and exit the program, and when relaunched, Israel is still highlighted.  The only way I can see to stop any words from being highlighted is to search for a nonsense term like "hhhhhh"

There doesn't seem to be any way to completely reset the search (to null basically). Whatever you last searched for always persists. 

It's not a huge deal but a clear function of some sort would be nice

Thank you

**Comment by tobias-klein** [comment]
Thanks for reporting this. So, this is about the "tab search" (searching within the currently opened tab), right? (as opposed to the module search function, which searches the whole Bible module)

I cannot fully reproduce this. When I delete the search term (remove it from the input field) the search updates accordingly and all highlights are removed as well.

I may need more details to reproduce this. Which Bible translation are you using? Which book did you open when using the search function?

**Comment by 0x199x** [comment]
Correct, the issue I'm having is with the tab search.  I am using master branch built from source (none of the AUR packages worked for me due to [alerque/aur#23](https://github.com/alerque/aur/issues/23))

On my end, when I delete the search term, nothing happens


https://user-images.githubusercontent.com/89231731/216776088-766e34ad-f875-4194-b82c-967646aef341.mp4

**Comment by tobias-klein** [comment]
Mmmm... ok, I see. You are talking about the "module search function" :).
Right now, the module search is not meant to be "reset" in the way you describe it. It works like the other content lookup functions (opening a book or opening a tagged verse list). When you open a book, perform a module search or open a tagged verse list the content of the current tab is always replaced.

If you don't want to loose the content of the current tab, you would need to first open a new tab and then perform the search there. Or alternatively (in the latest version on `master`) you can set the option to open search results in a dialog.

If you want to search within the currently opened book you can use the "tab search" to do an in-place search in the text of the current tab. Consider the option "Show current tab search".
![image](https://user-images.githubusercontent.com/17739895/216786759-42fc77dc-46ab-4645-8203-1dc46316e5d5.png)
(bottom right in the screenshot)

One thing that could help in the future is the possibility to navigate back to the previous content. I had opened an issue for that (see #827), but have not made any plans about implementing it yet.
